exercises/the-dice/canonical-data.json

```diff
@@ -0,0 +1,41 @@
+{
+  "exercise": "the-dice",
```

Comment:
Aside: two sixes is called [Boxcars](http://mathworld.wolfram.com/Boxcars.html), but I don't think that's as recognizable as snake eyes, so +1 to the suggested number swap.